### PR TITLE
pr-template: update link to contributing guidelines

### PR DESCRIPTION
**Pull request by jmbaur**
### Description

Current link to contributing guidelines is out of date.

### Checklist

- [ ] Change is backwards compatible.

- [ ] Code formatted with `./format`.

- [ ] Code tested through `nix-shell --pure tests -A run.all` or `nix develop --ignore-environment .#all` using Flakes.

- [ ] Test cases updated/added. See [example](https://github.com/nix-community/home-manager/commit/f3fbb50b68df20da47f9b0def5607857fcc0d021#diff-b61a6d542f9036550ba9c401c80f00ef).

- [x] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

    See [CONTRIBUTING](https://github.com/nix-community/home-manager/blob/master/docs/contributing.adoc#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.

- If this PR adds a new module

  - [ ] Added myself as module maintainer. See [example](https://github.com/nix-community/home-manager/blob/068ff76a10e95820f886ac46957edcff4e44621d/modules/programs/lesspipe.nix#L6).

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ pull-request.
 
 Also make sure to read the guidelines found at
 
-  https://github.com/nix-community/home-manager/blob/master/docs/contributing.adoc#sec-guidelines
+  https://github.com/nix-community/home-manager/blob/master/docs/manual/contributing/guidelines.md#guidelines-sec-guidelines
```
Comment (rycee) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: Thanks! Perhaps even better with

```suggestion
  https://nix-community.github.io/home-manager/index.xhtml#sec-guidelines
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Also make sure to read the guidelines found at
     {long description}
     ```
 
-    See [CONTRIBUTING](https://github.com/nix-community/home-manager/blob/master/docs/contributing.adoc#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.
+    See [CONTRIBUTING](https://nix-community.github.io/home-manager/#sec-guidelines) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.
```
Comment (rycee) [inline .github/PULL_REQUEST_TEMPLATE.md:38]:
```suggestion
    See [CONTRIBUTING](https://nix-community.github.io/home-manager/#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (rycee) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: I think you edited the wrong place 🙂

Comment (jmbaur) [reply .github/PULL_REQUEST_TEMPLATE.md]: oh, duh!

Comment (jmbaur) [reply .github/PULL_REQUEST_TEMPLATE.md]: should be fixed

**Comment by rycee** [comment]
Thanks! Merged to master now 🙂

**Pull request merged**